helm/cfgov/values.yaml

```
@@ -179,6 +179,14 @@ cronJobs:
       - "clearsessions"
     restartPolicy: OnFailure
 
+  - name: "publishPages"
+    schedule: "* * * * *"
+    command:
+      - "publish_scheduled_pages"
```

I think the command here will need to be `django-admin` like with `clearsessions above`.

```suggestion
      - "django-admin"
```

---

`django-admin` is unaware of `publish_scheduled_pages` it seems. Should be `cfgov/manage.py`.

---

Hmm, both should work though, if `clearsessions` works in the other job.